channels/sig_analog.h

```diff
@@ -346,6 +347,12 @@ struct analog_pvt {
 	 * gives a positive reply.
 	 */
 	unsigned int callwaitcas:1;
+
+	/*!
+	 * \brief TRUE if a Call Waiting Deluxe action is currently pending.
+	 */
+	unsigned int callwaitdeluxepending:1;
```

Comment:
Above, you have callwaitingdelux but here you have callwaitdeluxepending.  Should this be callwaitingdeluxepending (adding the "ing")?

Reply:
For consistency, "I" think naming the variables the same way is a plus.  I also don't see removing the "ing" as being at all helpful.  Lastly, I would put both of the variables together (e.g. these are all of the variables for the call waiting deluxe).

Reply:
I don't feel strongly about the presence/absence of "ing" so I can add that in.

But the variables should not be grouped together regardless. One is a config setting and the other is internal. The existing call waiting variables are separated out by this distinction.